src/plugins/BehaviorSpecGenerator/BehaviorSpecGenerator.js

```diff
@@ -254,28 +267,237 @@ define([
                 if (componentTypeNames.hasOwnProperty(name)) {
                     violations.push({
                         node: node,
-                        message: 'Duplicated name [' + name + '] shared with ' + componentTypeNames[name]
+                        message: 'Duplicated component [' + name + '] shared with ' + componentTypeNames[name]
                     });
                 }
                 componentTypeNames[name] = this.core.getPath(node);
 
+                var constructors = this.core.getAttribute(node, 'constructors');
+                if (constructors == '') {
+                    violations.push({
+                        node: node,
+                        message: 'Constructors empty for' + componentTypeNames[name]
+                    });
+                }
+                var definitions = this.core.getAttribute(node, 'definitions');
+                if (definitions == '') {
+                    violations.push({
+                        node: node,
+                        message: 'Definitions empty for' + componentTypeNames[name]
+                    });
+                }
+                var forwards = this.core.getAttribute(node, 'forwards');
+                if (definitions == '') {
+                    violations.push({
+                        node: node,
+                        message: 'forwards empty for' + componentTypeNames[name]
```

Comment:
"Formatting... This will be resolved to forwards empty forMyComponent"

Do not understand what you mean by this.
Please explain.